src/ec/packages/ec-component-footer-standardised/footer-standardised.story.js

```diff
@@ -0,0 +1,16 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs } from '@storybook/addon-knobs';
```

Comment:
I think we can avoid importing the knobs as well as setting them in the decorator, we don't have knobs defined for this component, right?

Reply:
ok